src/CalcViewModel/UnitConverterViewModel.cpp

```diff
@@ -803,10 +833,26 @@ void UnitConverterViewModel::UpdateInputBlocked(_In_ const wstring& currencyInpu
     m_isInputBlocked = false;
     if (posOfDecimal != wstring::npos && IsCurrencyCurrentCategory)
     {
-        m_isInputBlocked = (posOfDecimal + static_cast<size_t>(m_currencyMaxFractionDigits) + 1 == currencyInput.length());
+        m_isInputBlocked = (posOfDecimal + static_cast<size_t>(CurrencyFormatterFrom->FractionDigits) + 1 == currencyInput.length());
     }
 }
 
+void UnitConverterViewModel::UpdateCurrencyFormatter()
```

Comment:
Should we disable the '.' keyboard key in the case of yen too?

Reply:
I think this question is related to the first one -- Shall we keep the trailing decimal?
If we decided to show the trailing decimal, it would be weird to disable the '.' key.

Reply:
I am fine with keeping the existing behavior